### PR TITLE
Issue 1 create entry

### PR DESCRIPTION
**Pull request by tam-n**

### CODE REVIEW

**Comment by tam-n** [comment]
Confirmed data model is being saved to local storage and entryId is being incremented
image
image

**Comment by tam-n** [comment]
Desktop and mobile layouts
image
image

File: index.html [hunk 1]
```diff
@@ -14,14 +14,48 @@
...
   <main class="container">
-    <div data-view="entry-form"></div>
+
+    <div data-view="entry-form">
+
+      <form id="new-entry">
+        <h2>New Entry</h2>
+        <div class="row">
+          <div class="column-full column-half">
+            <img class="entry-image" src="images/placeholder-image-square.jpg">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: js/data.js [hunk 2]
```diff
@@ -6,3 +6,13 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+addEventListener('beforeunload', function (event) {
```
Comment (shawnkost) [inline js/data.js]: `addEventListener` is a method that needs to be attached to an object. You should be calling this on the `window` object

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+$inputImage.addEventListener('input', function (event) {
+  $entryImage.setAttribute('src', $inputImage.value);
+});
+
+const $form = document.querySelector('#new-entry');
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  const newEntry = {};
+  newEntry.entryId = data.nextEntryId;
```
Comment (shawnkost) [inline js/main.js]: You should have 3 other properties inside this object as well. We should be storing all of the form data inside this object, along with the id

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  data.nextEntryId++;
+  data.entries.push(newEntry);
```
Comment (shawnkost) [inline js/main.js]: `newEntry` should be getting added to the beginning of the array

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Tam! A few changes requested below. Here's one more thing I noticed

1. The image and "Notes" text / textarea should be aligned with the "New Entry" & "Code Journal" text. Basically all of your content should be aligned and start at the same place.

Let me know if you have any questions!

**Comment by tam-n** [comment]
Fixed Header alignment
image
image

File: js/main.js [hunk 5]
```diff
@@ -9,10 +9,15 @@ const $form = document.querySelector('#new-entry');
 
 $form.addEventListener('submit', function (event) {
   event.preventDefault();
-  const newEntry = {};
+  const newEntry = {
+    title: null,
+    photoURL: null,
+    notes: null,
+    entryId: null
```
Comment (shawnkost) [inline js/main.js]: These shouldn't be `null`. You should be storing the `input`s values from your `form`

File: js/main.js [hunk 6]
```diff
@@ -9,10 +9,15 @@ const $form = document.querySelector('#new-entry');
...
+  };
   newEntry.entryId = data.nextEntryId;
```
Comment (shawnkost) [inline js/main.js]: No need to set `entryId` to `null` here, you can just do `entryId: data.nextEntryId`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Tam!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**